modules/tinymce/src/core/main/ts/init/Init.ts

```diff
@@ -175,18 +176,17 @@ const init = (editor: Editor) => {
   initPlugins(editor);
   const renderInfo = renderThemeUi(editor);
   augmentEditorUiApi(editor, Optional.from(renderInfo.api).getOr({}));
-  const boxInfo = {
```

Comment:
Moved this as it's only needed in the iframe case, so initializing it here wasn't necessary and saved having to define a type.